lib/Common/Memory/Recycler.cpp

```diff
@@ -160,6 +164,9 @@ Recycler::Recycler(AllocationPolicyManager * policyManager, IdleDecommitPageAllo
     enableConcurrentMark(false),  // Default to non-concurrent
     enableParallelMark(false),
     enableConcurrentSweep(false),
+#if ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP
+    allocationsDuringConcurrentSweepTurnedOn(false),
```

Comment:
As I understand this, this is whether it's turned on for a particular collection, not for the thread context lifetime, correct? Would be good to rename this to make that clear, otherwise i dont understand whats the difference between this and the config flag

Reply:
Yes, this applies only for the current GC. I will rename the field to indicate that.

---
In reply to: [148168140](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148168140) [](ancestors = 148168140)